tests/integration/test_access_for_functions/test.py

```diff
@@ -76,4 +76,9 @@ def test_ignore_obsolete_grant_on_database():
     )
     instance.start_clickhouse()
 
-    assert instance.query("SHOW GRANTS FOR X") == "GRANT SELECT ON mydb.* TO X\n"
+    assert (
+        instance.query(f"SHOW GRANTS FOR `{user_id}`")
+        == f"GRANT SELECT ON mydb.* TO `{user_id}`\n"
+    )
+    instance.stop_clickhouse()
+    instance.start_clickhouse()
```

Comment:
Why do we need to stop/start clickhouse here?

Reply:
Not necessary now as I started to use different user names instead.

Reply:
I can do it as a follow up PR. I just do not want to trigger the whole CI for this trivial change as other PR is waiting.

Reply:
https://github.com/ClickHouse/ClickHouse/pull/68007